isis/CMakeLists.txt

```diff
@@ -309,16 +309,12 @@ if(APPLE)
   find_package(OpenGL REQUIRED)
 endif(APPLE)
 
-if(pybindings)
- find_package(Python REQUIRED)
-endif()
-
 # Iterate through all variables and extract the libraries and include directories
 get_cmake_property(_variableNames VARIABLES) # Get All VARIABLES
 
 set(ALLLIBDIRS "")
 set(ALLLIBS "")
-set(ALLINCDIRS "")
+set(ALLINCDIRS "${CMAKE_SOURCE_DIR}/inc")
```

Comment:
What does this change? Was this needed purely due to the python change?

Reply:
iirc the bindings were not able to find the headers, I'll look more into it to see if there is a better fix for it.

Reply:
If it works it works, just curious because cmake feels like a black box sometimes